modules/ROOT/pages/clustering/monitoring/endpoints.adoc

```diff
@@ -177,21 +178,30 @@ The following table explains how results can be compared.
+                                                                                                                                                                                                            Whenever any primaries are false that indicates a larger problem.
+| `allVotingSetsAreEqual`                    | For any 2 primaries (A and B), status endpoint A's `votingMembers`== status endpoint B's `votingMembers`.                                                          | When the voting begins, all the primaries are equal to each other, and all members agree on membership.
+| `allVotingSetsContainAtLeastTargetCluster` | For all primaries (*S*), excluding primary Z (to be switched off), every member in *S* contains *S* in their voting set.
+                                             Membership is determined by using the `memberId` and `votingMembers` from the status endpoint.                                                                   | Sometimes network conditions are not perfect and it may make sense to switch off a different primary than the one originally was to be switched off.
+                                                                                                                                                                                                            If this check is run for all primaries, the ones that match this condition can be switched off (providing other conditions are also met).
+| `hasOneLeader`                             | For any 2 primaries (A and B), `A.leader == B.leader && leader!=null`.                                                                                             | If the leader is different then there may be a partition (alternatively, this could also occur due to bad timing).
                                                                                                                                                                                                             If the leader is unknown, that means the leader messages have actually timed out.
-| `noMembersLagging`                         | For Primary A with `lastAppliedRaftIndex` = `min`, and Primary B with `lastAppliedRaftIndex` = `max`, `B.lastAppliedRaftIndex-A.lastAppliedRaftIndex<raftIndexLagThreshold`. | If there is a large difference in the applied indexes between Primaries, then it could be dangerous to switch off a Primary.
+| `noMembersLagging`                         | For primary A with `lastAppliedRaftIndex` = `min`, and primary B with `lastAppliedRaftIndex` = `max`, `B.lastAppliedRaftIndex-A.lastAppliedRaftIndex<raftIndexLagThreshold`. | If there is a large difference in the applied indexes between primaries, then it could be dangerous to switch off a primary.
 |===
 
+[NOTE]
+====
+`raftIndexLagThreshold` helps you to monitor the lag in applying Raft log entries across a cluster and set appropriate thresholds.
```

Comment:
I would add something stronger to point out that `raftIndexLagThreshold` is something the user defines, not something from the system, which is what confused Umar. For example:
> You should pick a `raftIndexLagThreshold` appropriate to your particular cluster and workload. Measuring the reported lag under normal circumstances and selecting a threshold slightly above that would be a good way to select an appropriate value.